MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include LICENSE
+include README.md
+include setup.cfg
+recursive-include pythoncz *
+recursive-include pythoncz_tests *
```

Comment:
Tohle nakopíruje do sdistu i `__pycache__` složky, podle mě je `*` příliš benevolentní. Pythonní modul by to mělo includovat samo, vše ostatní doporučuju whitelistovat.

```console
$ python setup.py sdist
...
copying pythoncz/__pycache__/__init__.cpython-34.pyc -> pythoncz-0.0.0/pythoncz/__pycache__
copying pythoncz/__pycache__/__init__.cpython-35.pyc -> pythoncz-0.0.0/pythoncz/__pycache__
copying pythoncz/__pycache__/__init__.cpython-36.pyc -> pythoncz-0.0.0/pythoncz/__pycache__
copying pythoncz/__pycache__/views.cpython-34.pyc -> pythoncz-0.0.0/pythoncz/__pycache__
copying pythoncz/__pycache__/views.cpython-35.pyc -> pythoncz-0.0.0/pythoncz/__pycache__
copying pythoncz/__pycache__/views.cpython-36.pyc -> pythoncz-0.0.0/pythoncz/__pycache__
...
copying pythoncz/models/__pycache__/__init__.cpython-34.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/__init__.cpython-35.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/__init__.cpython-36.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/beginners.cpython-34.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/beginners.cpython-35.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/beginners.cpython-36.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/github.cpython-34.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/github.cpython-35.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/github.cpython-36.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/jobs.cpython-34.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/jobs.cpython-35.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/jobs.cpython-36.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/photos.cpython-34.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/photos.cpython-35.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
copying pythoncz/models/__pycache__/photos.cpython-36.pyc -> pythoncz-0.0.0/pythoncz/models/__pycache__
...
```

Reply:
Co se tam tak běžně dává? `*.py` a zvlášť ne-py adresáře?